sandbox-templates/mdx-evaluate-create-react-app/src/index.js

```diff
@@ -0,0 +1,12 @@
+import { StrictMode } from "react";
+import ReactDOM from "react-dom";
+
+import App from "./App";
```

Comment:
Extension?

Reply:
:man_shrugging: works fine without it in CRA 4 and 5

Reply:
updated anyway